docs/pages/_meta.js

```diff
@@ -1,13 +1,19 @@
 export default {
-  index: "MUD",
-  blog: {
-    title: "Blog",
-    type: "page",
-    display: "hidden",
-  },
-  guides: "Guides",
-  tutorials: "Tutorials",
-  packages: "Packages",
+  index: "Introduction",
+  "quick-start": "Quick Start",
+  "what-is-mud": "What is MUD?",
+  "store": "Store",
+  "world": "World",
+  "mode": "MODE",
+  "cli": "CLI",
+  // blog: {
+  //   title: "Blog",
+  //   type: "page",
+  //   display: "hidden",
+  // },
```

Comment:
in my mind, blog posts are not evergreen and they're designed to be a snapshot in time, where the date stamp will be an indicator of whether or not something is no longer relevant (or we can add notices at the top of each post)

If we don't intend to have a blog on the docs site (I'm fine with this), we should at least take some care to redirect old URLs to the nearest relevant page in the new docs to not break backlinks, search engine results, etc.

Reply:
We can port the blog. The comments were from @alvrs exploration w/ Nextra.